CMakeLists.txt

```diff
@@ -597,6 +597,10 @@ set(Boost_USE_MULTITHREADED ON)
 find_package(Boost 1.61 COMPONENTS ${BOOST_COMPONENTS} REQUIRED)
 include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
 include_directories(SYSTEM ${PROJECT_BINARY_DIR}/include)
+if(FREEBSD)
+  list(REMOVE_ITEM Boost_LIBRARIES "/usr/local/lib/libboost_python.so")
```

Comment:
@tchaikov 
Make sense. Prevents dependany on what is embeded in the default stuff that comes with the packages. But note that I'm using the SYSTEM stuff, so I have no idea how that impacts your solution.